gcloud-java-dns/src/main/java/com/google/gcloud/dns/spi/DefaultDnsRpc.java

```diff
@@ -187,77 +221,127 @@ public Change getChangeRequest(String zoneName, String changeRequestId, Map<Opti
     }
   }
 
+  // todo(mderka) rename
+  private Dns.Changes.Get getChangeRequestRequest(String zoneName, String changeRequestId,
+      Map<Option, ?> options) throws IOException {
+    return dns.changes()
+        .get(this.options.projectId(), zoneName, changeRequestId)
+        .setFields(FIELDS.getString(options));
+  }
+
   @Override
   public ListResult<Change> listChangeRequests(String zoneName, Map<Option, ?> options)
       throws DnsException {
-    // options are fields, page token, page size, sort order
     try {
-      Dns.Changes.List request = dns.changes().list(this.options.projectId(), zoneName)
-          .setFields(FIELDS.getString(options))
-          .setMaxResults(PAGE_SIZE.getInt(options))
-          .setPageToken(PAGE_TOKEN.getString(options));
-      if (SORTING_ORDER.getString(options) != null) {
-        // todo check and change if more sorting options are implemented, issue #604
-        request = request.setSortBy(SORT_BY).setSortOrder(SORTING_ORDER.getString(options));
-      }
-      ChangesListResponse response = request.execute();
+      ChangesListResponse response = listChangeRequestsRequest(zoneName, options).execute();
       return of(response.getNextPageToken(), response.getChanges());
     } catch (IOException ex) {
       throw translate(ex);
     }
   }
 
+  // todo(mderka) rename
+  private Dns.Changes.List listChangeRequestsRequest(String zoneName, Map<Option, ?> options)
+      throws IOException {
+    // options are fields, page token, page size, sort order
+    Dns.Changes.List request = dns.changes().list(this.options.projectId(), zoneName)
+        .setFields(FIELDS.getString(options))
+        .setMaxResults(PAGE_SIZE.getInt(options))
+        .setPageToken(PAGE_TOKEN.getString(options));
+    if (SORTING_ORDER.getString(options) != null) {
+      // todo check and change if more sorting options are implemented, issue #604
+      request = request.setSortBy(SORT_BY).setSortOrder(SORTING_ORDER.getString(options));
+    }
+    return request;
+  }
+
   @Override
   public BatchRequest createBatch() {
     return dns.batch();
   }
 
   @Override
-  public BatchRequest prepareListZones(BatchRequest batch, JsonBatchCallback callback,
-      Map<DnsRpc.Option, ?> options) {
+  public BatchRequest addToBatchListZones(Object batch,
+      DnsRpc.Callback<ManagedZonesListResponse> callback, Map<DnsRpc.Option, ?> options) {
+    BatchRequest casted = (BatchRequest) batch;
```

Comment:
Here and elsewhere: I believe that the past participle of cast is still cast. Casted in an archaic form. Maybe `batchRequest` would be a better name for this variable.